### PR TITLE
Update the home page

### PR DESCRIPTION
**Pull request by kakulisen**
Add the toolkit release 0.1.0 information and reorganize Latest News

### CODE REVIEW

File: _pages/cn/home.md [hunk 1]
```diff
@@ -18,34 +18,35 @@ intro:
...
-    <a href='/cn/release/java-chassis'>Apache ServiceComb Java-Chassis 发布1.2.1</a>
+    <a href='/cn/release/pack-downloads'>Apache ServiceComb Pack 发布0.5.0</a>
   </li>
   <li>
-    <a href='/cn/docs/mersher-quick-start/'>Apache ServiceComb 新项目 Mesher </a>
+    <a href='/cn/release/toolkit-downloads'>Apache ServiceComb Toolkit 发布0.1.0</a>
   </li>
   <li>
-    <a href=''>Apache ServiceComb 荣获中国开源云联盟组织的中国优秀开源项目一等奖</a>
+    <a href='/cn/release/java-chassis'>Apache ServiceComb Java-Chassis 发布1.2.1</a>
```
Comment (MabinGo) [inline _pages/cn/home.md]: path of '/cn/release/java-chassis' is incorrect

File: _pages/home.md [hunk 2]
```diff
@@ -17,33 +17,34 @@ intro:
...
-    <a href='/release/java-chassis'>Apache ServiceComb Java-Chassis Release Version 1.2.1</a>
+    <a href='/release/pack-downloads'>Apache ServiceComb Pack 0.5.0 is released</a>
   </li>
   <li>
-    <a href='/docs/mersher-quick-start/'>Apache ServiceComb new project Mesher </a>
+    <a href='/release/toolkit-downloads'>Apache ServiceComb Toolkit 0.1.0 is released</a>
   </li>
   <li>
-    <a href=''>Apache ServiceComb won the first prize of China's outstanding open source project organized by China Open Source Cloud Alliance</a>
+    <a href='/release/java-chassis'>Apache ServiceComb Java-Chassis Release Version 1.2.1</a>
```
Comment (MabinGo) [inline _pages/home.md]: path of '/release/java-chassis' is incorrect

File: _pages/home.md [hunk 3]
```diff
@@ -17,33 +17,34 @@ intro:
   - excerpt: "
 <div class='excerpt-title'>Latest News</div>
 
+<style> li{width:100%} </style>
 <ul>
   <li>
-    <a href='/release/pack-downloads'>Apache ServiceComb Pack 0.5.0 is released</a>
+      <a href='/release/'>Apache ServiceComb Release Version 1.2.0</a>
```
Comment (MabinGo) [inline _pages/home.md]: '/release/' page is old, it should be updated

File: _pages/cn/home.md (hunk 1 above)
Comment (kakulisen) [reply _pages/cn/home.md]: thanks, I have fixed this problem in the new commit

File: _pages/home.md (hunk 2 above)
Comment (kakulisen) [reply _pages/home.md]: thanks, I have fixed this problem in the new commit

File: _pages/home.md (hunk 3 above)
Comment (kakulisen) [reply _pages/home.md]: thanks, I have fixed this problem in the new commit

File: _pages/cn/home.md (hunk 1 above)
Comment (kakulisen) [reply _pages/cn/home.md]: Thanks, I have fixed this problem in the new commit, pls help to check

File: _pages/home.md (hunk 2 above)
Comment (kakulisen) [reply _pages/home.md]: thanks, I have fixed this problem in the new commit, pls help to check

File: _pages/home.md (hunk 3 above)
Comment (kakulisen) [reply _pages/home.md]: thanks, I have fixed this problem in the new commit, pls help to check

**Comment by kakulisen** [comment]
I have fixed all the problems, please help to check again, thanks

**Comment by MabinGo** [comment]
LGTM

**Pull request merged**